visualizations/bar_chart/Makefile

```diff
@@ -9,7 +9,11 @@ dev-install:
 	pip install -e .
 
 install: build
+ifdef ARGS
+	pip install $(ARGS) .
+else
 	python setup.py install
```

Comment:
Maybe its best to default ARGS to empty string and only use `pip install $(ARGS) .`